flowcraft/generator/components/metagenomics.py

```diff
@@ -41,6 +41,42 @@ def __init__(self, **kwargs):
             "kraken"
         ]
 
+class Kraken2(Process):
+    """kraken2 process template interface
+
+            This process is set with:
+
+                - ``input_type``: fastq
+                - ``output_type``: txt
+                - ``ptype``: taxonomic classification
+    """
+    def __init__(self, **kwargs):
+
+        super().__init__(**kwargs)
+
+        self.input_type = "fastq"
+        self.output_type = "txt"
```

Comment:
The kraken nextflow process doesn't have a mai ``{{output_channel}}``, so it shouldn't have an output_type:

```suggestion
        self.output_type = None
```